BAZProjectC1/Components/TabBarMenu/TabBarMenu.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  TabBarMenu.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 21/09/22.
+//
+
+import UIKit
+
+class TabBarMenu: UITabBarController {
+
+    private var optionsMenu:[String] = ["Home", "Search"] {
+        didSet{
+            initOpcionsMenu()
+        }
+    }
+    
+    private var optionColor: UIColor = UIColor.appColorGraySecondary {
+        didSet{
+            initOpcionsMenu()
+        }
+    }
+    
+    private var selectionColor: UIColor = UIColor.appColorYellowPrimary {
+        didSet{
+            initOpcionsMenu()
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configurateElement()
+        initOpcionsMenu()
+    }
+    
+    func configurateElement() {
+        tabBar.backgroundColor = UIColor.appColorGrayPrimary
+        if #available(iOS 13, *) {
+            let appearance = UITabBarAppearance()
+            appearance.stackedLayoutAppearance.normal.titleTextAttributes = [NSAttributedString.Key.foregroundColor: optionColor]
+            appearance.stackedLayoutAppearance.selected.titleTextAttributes = [NSAttributedString.Key.foregroundColor: selectionColor]
+            tabBar.standardAppearance = appearance
+        }
+        let backgroundSize = CGSize(width: tabBar.frame.width / CGFloat(tabBar.items!.count), height: tabBar.frame.height)
```

Comment:
Evita hacer force unwrapp
```suggestion
        let backgroundSize = CGSize(width: tabBar.frame.width / CGFloat(tabBar.items?.count ?? 0), height: tabBar.frame.height)
```